cmake/modules/LLVM.cmake

```diff
@@ -16,7 +16,12 @@
 # under the License.
 
 # LLVM rules
-add_definitions(-DDMLC_USE_FOPEN64=0)
+# Due to LLVM debug symbols you can sometimes face linking issues on
+# certain compiler, platform combinations if you don't set NDEBUG.
+#
+# See https://github.com/imageworks/OpenShadingLanguage/issues/1069
+# for more discussion.
+add_definitions(-DDMLC_USE_FOPEN64=0 -DNDEBUG=1)
```

Comment:
It doesn't seem like a good idea to define `NDEBUG` globally.

Reply:
LLVM doesn't link correctly if you don't read the issues.

Reply:
```suggestion
add_definitions(-DDMLC_USE_FOPEN64=0)
target_add_definitions(tvm PRIVATE NDEBUG=1)
```

Reply:
On second pass this requires refactoring our CMake files, put a TODO in about.